CK2Plus_expanded/common/scripted_triggers/000_CK2Plus_expanded_scripted_triggers_religion.txt

```diff
@@ -36,4 +36,32 @@ is_celtic_secret_religious_society_member_trigger = {
 has_followers_of_fomorians_prerequisites = {
 	has_common_devilworship_prerequisites = yes
 	religion_openly_celtic_or_reformed_trigger = yes
+}
+
+religion_openly_basque_or_reformed_trigger = {
+	custom_tooltip = {
+		text = religion_openly_basque_or_reformed_tooltip
+		hidden_trigger = {
+			OR = {
+				religion = basque_pagan
+				religion = basque_pagan_reformed
+			}
+		}
+	}
+}
+
+true_religion_basque_pagan_trigger = {
+	true_religion = basque_pagan
+}
+
+true_religion_basque_pagan_reformed_trigger = {
+	true_religion = basque_pagan_reformed
+}
+

```

Comment:
The two above shouldn't be needed. In most places that you can call the trigger, you can just call `true religion` instead

Reply:
OK. I was just following what was written above by @Whizzer. Is that not necessary for Celtic either?

Reply:
I'm not entirely sure now. @Whizzer were there other circumstances for celtic that required a custom true_religion trigger instead of just using the `true_religion` trigger that's inclued?

Reply:
I merely copied what vanilla did, but you're right, we don't need those at all.

Reply:
We can leave this as-is then.

Reply:
Hmm, I am actually leaning toward keeping these in, so that we can make our religions as close to emulating vanilla as possible.

How do you both feel about that?

Reply:
Absolutely agreed.